### PR TITLE
Add pytest to runtime dependencies.

### PR DESCRIPTION
**Pull request by tobiasraabe**
### What problem do you want to solve?
 
Solves #71.

### CODE REVIEW

File: meta.yaml [hunk 1]
```diff
@@ -19,6 +18,7 @@ requirements:
   run:
     - python {{ python }}
     - pandas
+    - pytest
     - pyyaml
     - xlrd
```
Comment (hmgaudecker) [inline meta.yaml]: While we are at it -- do we still need the Excel reader?

Comment (tobiasraabe) [reply meta.yaml]: Yes. There is some test which needs excel files.

Comment (tobiasraabe) [reply meta.yaml]: It is because of `pensions.xlsx` use in `tax_transfer.py` and tests.

Comment (hmgaudecker) [reply meta.yaml]: Thanks. #73

**Pull request merged**